server/src/main.rs

```diff
@@ -29,9 +29,16 @@ async fn main() -> anyhow::Result<()> {
         .await
         .context("Error: Could not connect to mainnet database.")?;
 
+    let origins = [
+        "https://mina.vote/".parse::<HeaderValue>().unwrap(),
+        "http://99.79.130.169:8080/".parse::<HeaderValue>().unwrap(),
```

Comment:
I thought we agreed that we would get this value from an ENV variable and default to ANY?

Reply:
@jstavh This should be pulled from an ENV variable to make it more configurable across environments. If we don't set a value or it's empty, it should default to allowing `ANY` maintaining backwards compatibility.